frontend/src/router/pages/EditAndReview.jsx

```diff
@@ -39,57 +104,37 @@ function EditAndReview() {
             Bundle
           </label>
 
-          <select id="bundle" className="form-select">
-            <option value=""></option>
+          <select
+            id="bundle"
+            className="form-select"
+            value={bundleFilter}
+            onChange={(e) => setBundleFilter(e.target.value)}
+          >
+            <option value="">All</option>
+
+            {bundleValues.map((bundle) => (
+              <option key={bundle} value={bundle}>
+                {bundle}
+              </option>
+            ))}
           </select>
         </div>
 
         <div className="col-lg-3 col-md-6 col-12 d-flex">
-          <button className="btn btn-link align-self-end">Clear filters</button>
+          <button
+            onClick={resetFilters}
+            className="btn btn-link align-self-end"
+          >
+            Clear filters
+          </button>
         </div>
       </div>
 
-      <div className="table-responsive">
-        <table className="table table-striped">
-          <thead>
-            <tr>
-              <th scope="col">Park name</th>
-              <th scope="col">Status</th>
-              <th scope="col">Bundle</th>
-            </tr>
-          </thead>
-
-          <tbody>
-            <tr>
-              <th scope="row">Elk Falls Park</th>
-              <td>
-                <span className="badge rounded-pill text-bg-warning">
-                  Requested
-                </span>
-              </td>
-              <td>&nbsp;</td>
-            </tr>
-            <tr>
-              <th scope="row">Elk Lakes Park</th>
-              <td>
-                <span className="badge rounded-pill text-bg-warning">
-                  Requested
-                </span>
-              </td>
-              <td>&nbsp;</td>
-            </tr>
-            <tr>
-              <th scope="row">Elk Valley Park</th>
-              <td>
-                <span className="badge rounded-pill text-bg-success ">
-                  Approved
-                </span>
-              </td>
-              <td>&nbsp;</td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
+      {loading ? (
+        <p>Loading...</p>
```

Comment:
I'll ask Amanda if we have specs for a loading status, otherwise I'll just pop in a typical spinner thing.

Reply:
Just FYI, we used this animated progress bar: https://digitalspace.github.io/bcparks-bootstrap-theme on the BC Parks Find a Park page https://bcparks.ca/find-a-park, but of course, we don't have to use the same one both on public and internal sites.

Reply:
I chatted with Amanda today and she said that bar should be fine for our purposes. I'll add it in a separate branch and replace this "Loading..." text 👍

Reply:
Updated this branch with a `LoadingBar` component after discussing with Amanda 👍